src/components/parameter/FieldSelector.js

```diff
@@ -42,28 +43,18 @@ const FieldSelector = ({ schema, onChange, disabled, id }) => {
   const { t } = useTranslation();
   const classes = useStyles();
 
-  if (!schema || !schema.properties) {
+  if (!schema) {
```

Comment:
What if `properties` is null?

Reply:
Same as before, I included internal checks to deal with that possibility when the schema refers to an object